internal/servers/servers.go

```diff
@@ -50,7 +50,11 @@ func New(opts ...Option) Listener {
 	}
 
 	if l.sus != nil && len(l.sus) != 0 && (l.sds == nil || len(l.sds) == 0) {
-		copy(l.sds, l.sus)
+		s := make([]string, 0, len(l.sus))
```

Comment:
I changed because copying to arrays with length and cap of 0 is invalid

Reply:
@hlts2 
I think  the below code is more simple.
```golang
s := make([]int, len(l.sus)
copy(s, l.sus)
l.sds = s
```

is it possible ?
And, is there anything reason about your implement ?

Reply:
I'm sorry for being late.:bow:
Thank you! Fixed!